src/ui/widgets/EmbeddedDisplay/embeddedDisplay.tsx

```diff
@@ -40,41 +41,56 @@ export const EmbeddedDisplay = (
 
   log.debug(description);
 
-  // Apply the height to the top level if relative layout and none have been provided
-  if (props.position instanceof RelativePosition) {
-    props.position.height = props.position.height || description.height;
-    props.position.width = props.position.width || description.width;
-  }
+  // Check whether to override OPI autoZoomToFit
```

Comment:
Please elaborate in this comment why/how that would be done as well - and can this property be named so as to make it clearer what a value of true or false would do? i.e. we are overriding it but in which sense?